pkg/controller/broker/v2alpha4/activemqartemis/activemqartemis_reconciler.go

```diff
@@ -0,0 +1,1574 @@
+		}
+		volumeMount := volumes.MakeVolumeMount(volumeMountName)
+		volumeMounts = append(volumeMounts, volumeMount)
+	}
+
+	// Scan connectors for any with sslEnabled
+	for _, connector := range cr.Spec.Connectors {
+		if !connector.SSLEnabled {
+			continue
+		}
+		volumeMountName := cr.Name + "-" + connector.Name + "-secret-volume"
+		if "" != connector.SSLSecret {
+			volumeMountName = connector.SSLSecret + "-volume"
+		}
+		volumeMount := volumes.MakeVolumeMount(volumeMountName)
+		volumeMounts = append(volumeMounts, volumeMount)
+	}
+
+	if cr.Spec.Console.SSLEnabled {
+		volumeMountName := secrets.ConsoleNameBuilder.Name() + "-volume"
+		if "" != cr.Spec.Console.SSLSecret {
+			volumeMountName = cr.Spec.Console.SSLSecret + "-volume"
+		}
+		volumeMount := volumes.MakeVolumeMount(volumeMountName)
+		volumeMounts = append(volumeMounts, volumeMount)
+	}
+
+	return volumeMounts
+}
+
+func NewPodTemplateSpecForCR(customResource *brokerv2alpha4.ActiveMQArtemis) corev1.PodTemplateSpec {
+
+	// Log where we are and what we're doing
+	reqLogger := log.WithName(customResource.Name)
+	//reqLogger.Info("Creating new pod template spec for custom resource")
+	reqLogger.V(1).Info("NewPodTemplateSpecForCR")
+
+	namespacedName := types.NamespacedName{
+		Name:      customResource.Name,
+		Namespace: customResource.Namespace,
+	}
+
+	terminationGracePeriodSeconds := int64(60)
+
+	pts := pods.MakePodTemplateSpec(namespacedName, selectors.LabelBuilder.Labels())
+	Spec := corev1.PodSpec{}
+	Containers := []corev1.Container{}
+	container := containers.MakeContainer(customResource.Name, customResource.Spec.DeploymentPlan.Image, MakeEnvVarArrayForCR(customResource))
+	container.Resources = customResource.Spec.DeploymentPlan.Resources
+
+	volumeMounts := MakeVolumeMounts(customResource)
+	if len(volumeMounts) > 0 {
+		reqLogger.V(1).Info("Adding new mounts to main", "len", len(volumeMounts))
+		container.VolumeMounts = volumeMounts
+	}
+	reqLogger.V(1).Info("now mounts added to container", "new len", len(container.VolumeMounts))
+
+	Spec.Containers = append(Containers, container)
+	brokerVolumes := MakeVolumes(customResource)
+	if len(brokerVolumes) > 0 {
+		Spec.Volumes = brokerVolumes
+	}
+	Spec.TerminationGracePeriodSeconds = &terminationGracePeriodSeconds
+
+	//address settings
+	addressSettings := customResource.Spec.AddressSettings.AddressSetting
+	if len(addressSettings) > 0 {
+		//User has supplied addressSettings
+		envVarTuneFilePath := "TUNE_PATH"
+		outputDir := "/yacfg_etc"
+		envVarApplyRule := "APPLY_RULE"
+		envVarApplyRuleValue := customResource.Spec.AddressSettings.ApplyRule
+
+		if envVarApplyRuleValue == nil {
+			envVarApplyRuleValue = &defApplyRule
+		}
+		reqLogger.V(1).Info("Process addresssetting", "ApplyRule", *envVarApplyRuleValue)
+
+		brokerYaml := cr2jinja2v2alpha4.MakeBrokerCfgOverrides(customResource, nil, nil)
+
+		//resolve initImage
+		initImage := "registry.redhat.io/amq7/amq-broker-init-rhel7:0.2"
```

Comment:
Can you use quay.io/artemiscloud/activemq-artemis-broker-init?

Reply:
sure, it should be referring to artemiscloud, thanks!